torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3303,24 +3303,38 @@ def sample_inputs_conv2d(op_info, device, dtype, requires_grad, jit_fail_sample=
 def sample_inputs_group_norm(opinfo, device, dtype, requires_grad, **kwargs):
     make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
 
-    # Ordered as input shape, num groups, and eps
+    # Ordered as input shape, num groups, and kwargs for eps
```

Comment:
Also add some reference inputs for group norm. Instead of trying to exhaustively enumerate input combinations, `yield from` the sample inputs and then add a few larger hand-written samples